tests/resources/conformance_suites/nl_nt16/br_kvk/3.01-missing.xbrl

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Are these files duplicated between `tests/resources/validation/` and `tests/resources/conformance_suites/`?